source/kernel/Makefile

```
@@ -64,19 +53,17 @@ endef
 ###  MAIN RULE  ###
 ###################
 
-all: base ide ide-masteronly ide-emu ide-masteronly-emu ascii8 ascii16 mfrsd flashjacks ocm
+all: prerequisites base ide ide-masteronly ide-emu ide-masteronly-emu ascii8 ascii16 mfrsd flashjacks ocm
```

checkmake (0.3.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)